boards/arm/nrf52_adafruit_feather/nrf52_adafruit_feather.dts

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright (c) 2018 LEDCity AG
+ * Copyright (c) 2017 Linaro Limited
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/dts-v1/;
+#include <nordic/nrf52832.dtsi>
+
+/ {
+	model = "nRF52 Adafruit Feather";
+	compatible = "adafruit,nrf52_adafruit_feather", "nordic,nrf52832-qfaa",
+		     "nordic,nrf52832";
+
+	chosen {
+		zephyr,console = &uart0;
+#ifdef CONFIG_MCUMGR_SMP_UART
+		zephyr,uart-mcumgr = &uart0;
+#endif
+		zephyr,sram = &sram0;
+		zephyr,flash = &flash0;
+	};
+};
+
+&gpiote {
+	status ="ok";
+};
+
+&gpio0 {
+	status ="ok";
+};
+
+&uart0 {
+	compatible = "nordic,nrf-uart";
+	current-speed = <115200>;
+	status = "ok";
+};
+
+&i2c0 {
+	sda-pin = <25>;
+	scl-pin = <26>;
+};
+
+&flash0 {
```

Comment:
Why are those partitions different from the ones in:
https://github.com/zephyrproject-rtos/zephyr/blob/master/boards/arm/nrf52_pca10040/nrf52_pca10040.yaml

Reply:
Was originally copied from Blenano2 board (https://github.com/zephyrproject-rtos/zephyr/blob/master/boards/arm/nrf52_blenano2/nrf52_blenano2.dts). But I will change now to the nRF52_pca10040 partitions.